src/renderer/views/Post/Post.vue

```diff
@@ -28,4 +28,71 @@
   </div>
 </template>
 
-<script src="./Post.js" />
+<script setup>
+import { computed, onMounted, ref, watch } from 'vue'
+import FtCard from '../../components/ft-card/ft-card.vue'
+import FtCommunityPost from '../../components/FtCommunityPost/FtCommunityPost.vue'
+import FtLoader from '../../components/ft-loader/ft-loader.vue'
+import WatchVideoComments from '../../components/watch-video-comments/watch-video-comments.vue'
+import store from '../../store/index'
+import { useRoute, useRouter } from 'vue-router/composables'
+import { getInvidiousCommunityPost } from '../../helpers/api/invidious'
+
+const router = useRouter()
+const route = useRoute()
+
+const id = ref('')
+const authorId = ref('')
+const post = ref(null)
```

Comment:
As only the post itself will change but the data inside the posts won't change, we don't need deep reactivity and can switch to a `shallowRef` here.
```suggestion
const post = shallowRef(null)
```